public_html/templates/index.html

```diff
@@ -1,100 +1,118 @@
-                               class="btn btn-success btn-block"
-                               title="The edit was a copyright violation and has been reverted" value="Page fixed"
-                               disabled>
-                        <input type="button" id="{{ 'danger' ~ row.ithenticate_id }}"
-                               class="btn btn-danger-clicked btn-block"
-                               title="The edit is a false positive, nothing needs to be done" value="No action needed"
-                               onclick="saveState({{ row.ithenticate_id }}, 'false')">
-                    {% elseif row.status == NULL %}
-                        <input type="button" id="{{ 'success' ~ row.ithenticate_id }}"
-                               class="btn btn-success btn-outline btn-block"
-                               title="The edit was a copyright violation and has been reverted" value="Page fixed"
-                               onclick="saveState({{ row.ithenticate_id }}, 'fixed')">
-                        <input type="button" id="{{ 'danger' ~ row.ithenticate_id }}"
-                               class="btn btn-danger btn-block"
-                               title="The edit is a false positive, nothing needs to be done" value="No action needed"
-                               onclick="saveState({{ row.ithenticate_id }}, 'false')">
-                    {% endif %}
-                </div>
-            </div>
-            <div class="compare-links-container">
-                <a class="btn btn-xs btn-primary compare-button" href='{{ r }}' target="_blank">
-                    <span class="glyphicon glyphicon-new-window"></span>
-                    Turnitin report
-                </a><br>
-                {% if row.copyvios is not empty %}
-                    {% for copy in row.copyvios %}
-                        <button class="btn btn-xs btn-primary dropdown-toggle compare-button" onclick="">
-                            <span class="glyphicon glyphicon-chevron-down" aria-hidden="true"></span>
-                            Compare
-                        </button>
-                        <a href="{{ copy }}" target="_blank">{{ copy }}</a><br/>
-                        <div class="compare-div" id="{{ 'comp' }}">
-                            <div class="compare-edit compare-pane">
-                                Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Aenean commodo ligula eget
-                                dolor.
-                            </div>
-                            <div class="compare-website compare-pane">
-                                Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Aenean commodo ligula eget
-                                dolor.
-                            </div>
-                        </div>
-                    {% endfor %}
-                {% endif %}
             </div>
         {% endfor %}
     {% endif %}
```

Comment:
In short what happened here: I added a wrapper around the two divs that make up an entry in the interface, moving the padding and border styles to it, and indented everything in between. Much smaller change than it appears to be!